clang/test/Driver/fp-contract.c

```diff
@@ -10,18 +10,18 @@
 // RUN: %clang -### -fno-fast-math -c %s 2>&1 \
 // RUN:   | FileCheck --check-prefix=CHECK-FPC-ON %s
 
-// RUN: %clang -### -Werror -ffast-math -ffp-contract=on -c %s 2>&1 \
+// RUN: %clang -### -ffast-math -ffp-contract=on -c %s 2>&1 \
```

Comment:
If these RUN lines now emit warnings (upgraded to errors if -Werror), consider adding check prefixes for the warnings.